docs/about/contributing/getting-started-developing.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: main
+title: Getting Started Developing
+category: About
+menu: menu
+toc:
+    - title: Getting Started Developing
+      url: "#getting-started-developing"
+      active: true
+    - title: Running the Screwdriver API and UI
+      url: "#running-the-screwdriver-api-and-ui"
+    - title: Running SD-in-a-Box
+      url: "#running-sd-in-a-box"
+---
+# Getting Started Developing
+
+There's two different ways to set up your dev environment locally in order to run and test Screwdriver locally:
+
+1. **Run the Screwdriver API and UI locally** - ideal for testing out npm package dependencies that don't require a build to be run
+2. **Run SD-in-a-box** - ideal for running builds
+
+# Running the Screwdriver API and UI
+## Running the API
+
+### Prerequisites
+- Node v8.0.0 or higher
+- [Kubernetes][kubectl] or [Docker][docker]
+
+### Installing
+
+```bash
+$ git clone git@github.com:screwdriver-cd/screwdriver.git ./
+$ npm install
+# The API stores default configuration data in ./config/default.yaml.
+# You'll need to overwrite using your own config data in a local.yaml.
+$ vim ./config/local.yaml
+```
+
+### Configuring your local.yaml
+There are several fields you will need to overwrite:
+
+1. `auth`
+1. `httpd`
+1. `scms`
+1. `ecosystem`
```

Comment:
```
1. `auth`
1. `httpd`
1. `scms`
1. `ecosystem`
```

From my understanding Jekyll will automatically correct the enumeration, but it's confusing when reading the raw source.

Reply:
I think it's generally better to just put the 1s everywhere in the file so if people ever want to add something in the middle or rearrange the order, they don't have to individually update each number